core/qt-init.cpp

```diff
@@ -27,10 +27,31 @@ void init_qt_late()
 	// enable user specific settings (based on command line argument)
 	if (settings_suffix) {
 		if (verbose)
+#if defined(SUBSURFACE_MOBILE) && (defined(Q_OS_LINUX) && !defined(Q_OS_ANDROID)) || (defined(Q_OS_DARWIN) && !defined(Q_OS_IOS))
```

Comment:
in preprocessor conditions, does || bind stronger than && ? Otherwise this might not do what you want...